mitodl/ocw_oer_export
# Add CR_CREATE_DATE and update CR_SUBLEVEL, CR_COU_COPYRIGHT_HOLDER

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/3514

### Description (What does it do?)
Adds `CR_CREATE_DATE` and updates `CR_SUBLEVEL`, and `CR_COU_COPYRIGHT_HOLDER` using the instructions mentioned in the ticket and [ongoing discussion](https://github.com/mitodl/hq/issues/2426#issuecomment-1929859194).

We're still being updated on the exact months to use for semesters, but that would be a trivial change so this can be reviewed in the meantime.

### Updated CSV Link
https://docs.google.com/spreadsheets/d/1uBFHJD0LpMqR62JQdXrGMV8k4TQwW5FGz07GTtOAYL0/edit#gid=660003245

### How can this be tested?
Make sure the changes are working as expected in the above CSV link.

Now checkout to this branch, run the program, make sure everything is good:
1. Build the container:

```
docker compose build
```
2. Start the container:

```
docker compose run --rm app
```

3. To generate a JSON file containing complete API data:

```
docker compose run --rm app --create_json
```

4. To create a CSV file from the local JSON file:

```
docker compose run --rm app --create_csv --source=json
```

5. To run unit tests:

```
docker run --rm ocw_oer_export python -m unittest discover
```
